zeppelin-server/src/test/java/org/apache/zeppelin/integration/ParagraphActionsIT.java

```diff
@@ -120,6 +120,61 @@ public void testMoveUpAndDown() throws Exception {
+      createNewNote();
+
+      waitForParagraph(1, "READY");
+
+      collector.checkThat("Before Show Title : The title field contains",
+              driver.findElement(By.xpath(getParagraphXPath(1) + "//div[contains(@class, 'binding')]")).getText(),
+              CoreMatchers.equalTo(""));
+      driver.findElement(By.xpath(getParagraphXPath(1) + "//span[@class='icon-settings']")).click();
+      collector.checkThat("Before Show Title : The title option in option panel of paragraph is labeled as  ",
+              driver.findElement(By.xpath(getParagraphXPath(1) + "//a[contains(@ng-click, 'showTitle()')]")).getText(),
+              CoreMatchers.equalTo("Show title"));
+
+      driver.findElement(By.xpath(getParagraphXPath(1) + "//ul/li/a[@ng-click='showTitle()']")).click();
+      collector.checkThat("After Show Title : The title field contains",
+              driver.findElement(By.xpath(getParagraphXPath(1) + "//div[contains(@class, 'binding')]")).getText(),
+              CoreMatchers.equalTo("Untitled"));
+
+      driver.findElement(By.xpath(getParagraphXPath(1) + "//span[@class='icon-settings']")).click();
+      collector.checkThat("After Show Title : The title option in option panel of paragraph is labeled as",
+              driver.findElement(By.xpath(getParagraphXPath(1) + "//a[contains(@ng-click, 'hideTitle()')]")).getText(),
+              CoreMatchers.equalTo("Hide title"));
+
+      driver.findElement(By.xpath(getParagraphXPath(1) + "//ul/li/a[@ng-click='hideTitle()']")).click();
+      collector.checkThat("After Hide Title : The title field contains",
+              driver.findElement(By.xpath(getParagraphXPath(1) + "//div[contains(@class, 'binding')]")).getText(),
+              CoreMatchers.equalTo(""));
+      driver.findElement(By.xpath(getParagraphXPath(1) + "//span[@class='icon-settings']")).click();
+      driver.findElement(By.xpath(getParagraphXPath(1) + "//ul/li/a[@ng-click='showTitle()']")).click();
+
+      driver.findElement(By.xpath(getParagraphXPath(1) + "//div[contains(@class, 'title')]")).click();
+      driver.findElement(By.xpath(getParagraphXPath(1) + "//input")).sendKeys("NEW TITLE" + Keys.ENTER);
+      collector.checkThat("After Editing the Title : The title field contains ",
+              driver.findElement(By.xpath(getParagraphXPath(1) + "//div[contains(@class, 'binding')]")).getText(),
+              CoreMatchers.equalTo("NEW TITLE"));
+      ZeppelinITUtils.sleep(1000,false);
+      driver.navigate().refresh();
+      collector.checkThat("After Page Refresh : The title field contains ",
+              driver.findElement(By.xpath(getParagraphXPath(1) + "//div[contains(@class, 'binding')]")).getText(),
+              CoreMatchers.equalTo("NEW TITLE"));
+      deleteTestNotebook(driver);
+
+    } catch (Exception e) {
+      LOG.error("Exception in ParagraphActionsIT while testTitleButton ", e);
+      File scrFile = ((TakesScreenshot) driver).getScreenshotAs(OutputType.FILE);
+      throw e;
```

Comment:
throw e (Exception);  but signature throws InterruptedException, can you change that ?